docs_src/sql_databases/sql_app/crud.py

```diff
@@ -24,6 +24,23 @@ def create_user(db: Session, user: schemas.UserCreate):
     return db_user
 
 
+def update_user(db: Session, user_id: int, new_password: str):
+    db_user_to_update = db.query(models.User).filter(models.User.id == user_id).first()
```

Comment:
@tiangolo uses `db_user` across all the documentation, i think you should stay with it.